readers/elasticsearch/src/main/java/com/criteo/hadoop/garmadon/elasticsearch/ElasticSearchReader.java

```diff
@@ -111,6 +111,15 @@ void writeToES(GarmadonMessage msg) {
         } else {
             Map<String, Object> eventMap = ProtoConcatenator.concatToMap(Arrays.asList(msg.getHeader(), msg.getBody()), true);
             eventMap.put("event_type", msgType);
+
+            // Specific normalization for FS_EVENT
+            if ("FS_EVENT".equals(msgType)) {
+                String uri = (String) eventMap.get("uri");
+                eventMap.computeIfPresent("src_path", (k, v) -> ((String) v).replace(uri, ""));
```

Comment:
I'm guessing we disregard cases where uri might be in the middle of these strings (just checking)

Reply:
"hdfs://<nameservice>" or "viewfs://<namservice" should not, I think we can disregard